matRadGUI.m

```diff
@@ -1748,7 +1748,6 @@ function btnuiTableAdd_Callback(hObject, ~, handles)
 data{sEnd+1,2} = 'Select VOI Type';
 data{sEnd+1,3} = 2;
 data{sEnd+1,4} = 'Select obj func/constraint';
-data{sEnd+1,6} = '';
```

Comment:
would you modifiy your pull request accordingly? Then I am gonna merge
everything. Thanks!

On Thu, Jul 7, 2016 at 10:11 AM, Henning Mescher <notifications@github.com>
wrote:

> In matRadGUI.m
> <https://github.com/e0404/matRad/pull/84#discussion_r69866607>:
>
> > @@ -1748,7 +1748,6 @@ function btnuiTableAdd_Callback(hObject, ~, handles)
> >  data{sEnd+1,2} = 'Select VOI Type';
> >  data{sEnd+1,3} = 2;
> >  data{sEnd+1,4} = 'Select obj func/constraint';
> > -data{sEnd+1,6} = '';
>
> Yes.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/e0404/matRad/pull/84/files/ee230d992709006f8019767a784b04383b47aec8#r69866607>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJr8fV26Swbe0GtDvEnZFgv0tLyObC4Xks5qTLSogaJpZM4JGzNL>
> .
>